### PR TITLE
Fix filter-by-pr-status when no Pull Requests exist

### PR DESCRIPTION
**Pull request by busches**
This addresses the issue called out here: https://github.com/sindresorhus/refined-github/pull/1904#issuecomment-485615635

I didn't want to return on the first selector, as if GH changes the DOM, then we won't realize the feature broke as easily.

### CODE REVIEW

File: source/features/filter-pr-by-build-status.tsx [hunk 1]
```diff
@@ -41,6 +41,11 @@ function populateDropDown({currentTarget}: Event): void {
 }
 
 function init(): void {
+	// If there are no PRs then return
+	if (!select('.table-list-filters')) {
```
Comment (HardikModha) [inline source/features/filter-pr-by-build-status.tsx]: Prefer `select.exists`. We are following the same everywhere.

**Comment by bfred-it** [comment]
The error is on `reviewFilter`, we can just add the if there. Also it should return false.

File: source/features/filter-pr-by-build-status.tsx (hunk 1 above)
Comment (busches) [reply source/features/filter-pr-by-build-status.tsx]: Duh, I'll fix it.

**Comment by busches** [comment]
@bfred-it I opted for the secondary check since if GH changes markup the feature will be disabled instead of failing. Same risk with my fix, but it's a lot less specific. Can it return false if we have it typed as void?

**Comment by bfred-it** [comment]
> if GH changes markup the feature

We can't really do _anything_ about it. GH can change `details` just like it can change `.table-list-filters`, we can't say which one is more likely to happen. 

Lately I've been trying to avoid these `!exists` checks to let the errors through, but sometimes they're necessary like in this case.

**Comment by busches** [comment]
> > if GH changes markup the feature
> 
> We can't really do _anything_ about it. GH can change `details` just like it can change `.table-list-filters`, we can't say which one is more likely to happen.
> 
> Lately I've been trying to avoid these `!exists` checks to let the errors through, but sometimes they're necessary like in this case.

Alright, you've convinced me. :)

**Pull request merged**